### PR TITLE
AoU Echo Precision and Sensitivity Updates [VS-1093]

### PR DESCRIPTION
**Pull request by mcovarr**
P+S doc fixups for Delcho

### CODE REVIEW

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md [hunk 1]
```diff
@@ -80,6 +80,7 @@
     - This workflow transforms the data in the vet tables into a schema optimized for callset stats creation and for calculating sensitivity and precision.
     - It will need to be run twice. Once with `only_output_vet_tables` set to "true" (the default value is `false`) and once with `control_samples` set to "true" (the default value is `false`).
     - See [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix, which you will need to keep track of for the callset stats. Use the same prefix for both runs.
+    - **NOTE** Set `use_compressed_references` to true.
```
Comment (gbggrant) [inline scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:83]: We should consider having this workflow auto-detected that the references are compressed

Comment (mcovarr) [reply scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:83]: Totally agree, created VS-1138.

**Review by rsasch — CHANGES_REQUESTED** [review]
Can we link from the AOU_DELIVERABLES doc to the AoU_PRECISION_SENSITIVITY doc so that we only need to remember the location of one (the DELIVERABLES) doc?

Also, can you clean up the parts of the DELIVERABLES doc that are covered in the P&S doc (e.g. the runs of Prepare and Extract with `controls` set to true)?

File: scripts/variantstore/tieout/AoU_PRECISION_SENSITIVITY.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+1. Complete (at least) through the training step (`GvsCreateFilterSet`) of the [GVS pipeline](../docs/aou/AOU_DELIVERABLES.md) with all the samples (control and non-control).
+1. Run the `GvsPrepareRanges` WDL with the following input values:
+   1. `call_set_identifier`, `dataset_name`, `project_id` set to the same values as the inputs of the same name in the preceding `GvsCreateFilterSet` run.
+   1. `control_samples` set to `true`.
+   1. Make sure to set `use_compressed_references` to the appropriate value. This will be `true` for AoU callsets starting with Echo.
+1. Run the `GvsExtractCallset` WDL with the following input values:
+   1. `call_set_identifier`, `dataset_name`, `filter_set_name`, `project_id` set to the same values as the inputs of the same name in the preceding `GvsCreateFilterSet` run.
+   1. `extract_table_prefix` set to the same value as `call_set_identifier`.
+   1. `control_samples` set to `true`.
+   1. `scatter_count` set to 500. This is an optional input value but at least for Echo scale test control extract prior to P+S, leaving this unset causes most of the 111 `ExtractTaxk` shards to OOM.
```
Comment (RoriCremer) [inline scripts/variantstore/tieout/AoU_PRECISION_SENSITIVITY.md:13]: these damn 1.s drive me nuts!

Comment (RoriCremer) [inline scripts/variantstore/tieout/AoU_PRECISION_SENSITIVITY.md:13]:
```suggestion
   1. `scatter_count` set to 500. This is an optional input value but at least for the Echo scale test control extract prior to P+S, leaving this unset causes most of the 111 `ExtractTask` shards to OOM.
```

File: scripts/variantstore/tieout/AoU_PRECISION_SENSITIVITY.md [hunk 3]
```diff
@@ -16,13 +26,14 @@ The `GvsExtractCallset` workflow should have an output of type `Array[File]` cal
 gs://<long prefix with workspace id, submission id, etc>/call-ExtractTask/shard-0/control-001-extract.vcf.gz
 ```
 
-Then all the output control VCFs might be selected into a FOFN with some globbing:
+Then all the output control VCFs might be selected into a FOFN with some globbing. Perform the following in a notebook
+in the workspace:
```
Comment (RoriCremer) [inline scripts/variantstore/tieout/AoU_PRECISION_SENSITIVITY.md:30]: do we need to give any notebook set up info?

**Review by RoriCremer — APPROVED** [review]

**Review by rsasch — APPROVED** [review]
looks good — thanks for the changes!

**Pull request merged**